sdk/python/tests/test_historical_retrieval.py

```diff
@@ -454,13 +459,26 @@ def test_historical_features_from_bigquery_sources(
         actual_bq_temp_table = bigquery.Client().get_table(bq_temp_table_path)
         assert actual_bq_temp_table.table_id == bq_temp_table_path.split(".")[-1]
 
+        # Config to overwrite an existing table should succeed
+        job_config = bigquery.QueryJobConfig(
+            destination=path, write_disposition="WRITE_TRUNCATE"
```

Comment:
Note: [write_disposition](https://googleapis.dev/python/bigquery/latest/generated/google.cloud.bigquery.job.QueryJobConfig.html#google.cloud.bigquery.job.QueryJobConfig.write_disposition) has the string options listed in the docs [here](https://cloud.google.com/bigquery/docs/reference/rest/v2/Job#JobConfigurationQuery.FIELDS.write_disposition).

Reply:
Same as comment above, and we don't need to test for individual settings in QueryJobConfig. We can assume they work